homeassistant/components/airvisual/sensor.py

```diff
@@ -1,47 +1,61 @@
 """Support for AirVisual air quality sensors."""
 from logging import getLogger
 
+from homeassistant.components.air_quality import AirQualityEntity
```

Comment:
We're not allowed to import the air quality entity base class to a sensor platform.